src/test/a11y/a11y.ts

```diff
@@ -23,7 +23,7 @@ app.locals.csrf = 'dummy-token'
 
 const cookieName: string = config.get<string>('session.cookieName')
 
-const agent = supertest.agent(app)
+const agent = supertest(app)
```

Comment:
Instance created this way does not save cookies so they can be attached manually to requests.